apps/rush/tslint.json

```diff
@@ -50,7 +50,6 @@
     "no-trailing-whitespace": true,
     "no-unnecessary-semicolons": true,
     "no-unused-expression": true,
-    "no-unused-imports": true,
```

Comment:
It's built into TS now, so the rule has been removed.

---
In reply to: [129393749](https://github.com/Microsoft/web-build-tools/pull/276#discussion_r129393749) [](ancestors = 129393749)